src/app/public/modules/action-hub/action-hub.component.spec.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyActionHubComponent
+} from './action-hub.component';
+
+import {
+  Component,
+  Input
+} from '@angular/core';
+
+import {
+  Observable, of
+} from 'rxjs';
+
+import {
+  Configuration
+} from './types/configuration';
+
+import {
+  SkyActionHubModule
+} from './action-hub.module';
+
+// Test component
+
+@Component({
+  selector: 'skyux-action-hub-test',
+  template: `
+    <sky-action-hub
+      [config]="configAsync"
+    ></sky-action-hub>
+  `
+})
+class ActionHubTestComponent {
```

Comment:
Historically, we have placed these test components outside of the `spec` file in a `fixtures` folder. We do this to help de-clutter things and make it easier to isolate things if we need to make changes to the test components in the future.